substrate/frame/whitelist/src/lib.rs

```diff
@@ -44,22 +44,18 @@ extern crate alloc;
 
 use alloc::boxed::Box;
 use codec::{DecodeLimit, Encode, FullCodec};
-use frame_support::{
-	dispatch::{GetDispatchInfo, PostDispatchInfo},
-	ensure,
+use frame::{
+	deps::sp_api,
```

Comment:
Why this? `deps::sp_api`

Reply:
`sp_api::MAX_EXTRINSIC_DEPTH,`

Line 171